lib/Data/Validate/Sanctions.pm

```diff
@@ -28,8 +31,29 @@ sub new {    ## no critic (RequireArgUnpacking)
     return bless $self, ref($class) || $class;
 }
 
-my $sanction_file = _default_sanction_file();
-my $instance;
+sub update_data {
+    my $self = shift;
+
+    my $new_data = Data::Validate::Sanctions::Fetcher::run();
+    $self->_load_data();
+    
+    my $updated;
+    foreach my $k (keys %$new_data) {
+        if (ref($self->{_data}{$k}) ne 'HASH' || $self->{_data}{$k}{updated} < $new_data->{$k}{updated}) {
+            $self->{_data}{$k} = $new_data->{$k};
+            $updated = 1;
+        }
+    }
+
+    $self->_save_data if $updated;
+    return;
+}
+
+sub last_updated {
+    my $self = shift;
+    my $list = shift;
+    return $list ? $self->{_data}->{$list}->{updated} : $self->{last_time};
+}
```

Comment:
Why were these moved up?  Seems unneeded change here...

Reply:
In one of my old PRs (not sure which), @code4pay mentioned that private subs should be at the start

Reply:
Oh wait -.- my bad. they should be at the bottom, as per: https://github.com/binary-com/perl-Data-Validate-Sanctions/pull/23#discussion_r226208767 Making changes now